src/Bicep.Core/TypeSystem/Radius/CommonBindings.cs

```diff
@@ -57,6 +57,17 @@ public static class CommonBindings
             additionalPropertiesType: null,
             additionalPropertiesFlags: TypePropertyFlags.None);
 
+        public static NamedObjectType BindingRedis = new NamedObjectType(
+            "binding: redislabs.com/Redis",
+            validationFlags: TypeSymbolValidationFlags.WarnOnTypeMismatch,
+            properties: new []
+            {
+                new TypeProperty("kind", new StringLiteralType("redislabs.com/Redis"), TypePropertyFlags.Required),
+                new TypeProperty("connectionString", LanguageConstants.String, TypePropertyFlags.None),
```

Comment:
Did you end up adding any other values here? or just connection string?

Send another PR if you want to add more.